classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
```

Comment:
- [ ] As requested by the previous reviewer kindly add this line 👇 
```
student.classroom = self
```
to implement the belongs-to relationship between the student and classroom 👍